.github/actions/build-dotnet/Write-CacheConfiguration.ps1

```diff
@@ -1,6 +1,6 @@
 param($IsNuget, $IsNpm, $RestoreKeys, $Hash)
 
-function Set-Output($Key, $Value) { Write-Output "::set-output name=$Key::$Value" }
```

Comment:
We actually have a function for this? Great, please then refactor the rest of the code to use this one instead of the custom `Write-Output`s. Such common scripts are under the root Scripts folder, you can see how those are accessed.